### PR TITLE
Add sections for search link and empty states

### PR DESCRIPTION
**Pull request by bangelic**
Two new sections for Managing Content Views have been added. One is for
clearing the search link and the other is for
standardizing content
view empty states. The info was pulled from the demo video so there may
need to be some more content needed and also the
section for
standardizing content view empty states may need a new title since it
doesn't quite fit the context of the content.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.5/Katello 4.7
* [ ] Foreman 3.4/Katello 4.6
* [ ] Foreman 3.3/Katello 4.5
* [ ] Foreman 3.2/Katello 4.4
* [ ] Foreman 3.1/Katello 4.3
* For Foreman 3.0 or older, please create a separate PR.
* We do not accept PRs for Foreman 2.3 or older.

### CODE REVIEW

File: guides/common/modules/con_clearing-the-search-filter.adoc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[id="Clearing_the_Search_Filter{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/con_clearing-the-search-filter.adoc:1]:
```suggestion
[id="Clearing_the_Search_Filter_{context}"]
```

File: guides/common/modules/con_standardizing-content-view-empty-states.adoc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[id="Standardizing_Content_View_Empty_States{context}"]
```
Comment (maximiliankolb) [inline guides/common/modules/con_standardizing-content-view-empty-states.adoc:1]:
```suggestion
[id="Standardizing_Content_View_Empty_States_{context}"]
```

**Comment by bangelic** [comment]
@parthaa Can we get someone from Team Phoenix to review this PR?

**Comment by bangelic** [comment]
@maximiliankolb QE review done and ACK. Can this get merged?

**Review by maximiliankolb — APPROVED** [review]
LGTM!

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked to 3.5: :heavy_check_mark: 
   cf381d2e3..a96a5c656  3.5 -> 3.5

**Pull request merged**